packages/docs/app/manifesto/page.tsx

```diff
@@ -191,33 +182,28 @@ export default function ManifestoPage() {
             <p>
               Setting up powerful, resilient event-based microservice systems is incredibly difficult to do correctly.{' '}
               <span className={gradientTextBlue}>Motia builds this foundation for you</span>, providing the
-              necessary resiliency without requiring engineers to worry about the underlying complexity. They can focus
-              on building workflows.
+              necessary resiliency without requiring engineers to worry about the underlying complexity. They can instead focus on building workflows, and implementing business logic.
             </p>
           </section>
 
           <section className={normalText}>
             <p>
               We have <span className={gradientTextBlue}>25 years of knowledge</span> about event-based systems and
               microservices. We don't need new ways to connect workflows when we have strong patterns already
-              established. Motia uses this knowledge to build a unified system that joins the functions previously spread
-              across separate API servers, background systems, and AI agent frameworks.
+              established. Motia leverages this knowledge to build a unified system that joins the functions previously spread across disconnected silos - API servers, background systems, and AI agent frameworks.
             </p>
           </section>
 
           <section className={normalText}>
             <p>
-              <span className={gradientTextBlue}>Motia is built as a highly scalable enterprise solution</span>, solving
-              key problems that other systems miss. It tackles the hidden, but important, challenges that appear
-              when growing large codebases and system issues that are hard to truly understand without seeing them yourself.
+              Motia has been built from the ground up as <span className={gradientTextBlue}> a highly scalable enterprise solution</span>, solving
+              key problems that other systems miss. It addresses the hidden yet critical challenges that emerge as codebases grow, problems that are difficult to grasp without experiencing them firsthand yourself.
             </p>
           </section>
 
           <section className={normalText}>
             <p>
-            A developer-focused event-driven system is needed and will become a tool of choice. Whether Motia or another tool finally becomes the main
-              choice, a solution that brings these concerns together and gives a clear, developer-focused approach is
-              the natural next step in how software engineering will change with AI.{' '}
+              To navigage the AI-first world, developers need an event-driven system built with them in mind. A system that unifies AI, backend logic, and automation into a single, cohesive model{' '}
```

Comment:
Change this sentence back to the previous version; the previous one doesn't sound robotic and is not too much AI-oriented.